iml-integration-tests/tests/integration_test.rs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) 2019 DDN. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+mod vagrant;
+
+#[tokio::test]
+async fn test_setup() -> Result<(), Box<dyn std::error::Error>> {
+    let config = vagrant::ClusterConfig::default();
+
+    vagrant::setup_deploy_servers(&config, "base_managed_patchless").await?;
```

Comment:
Why a new wrapper binary in Rust instead of a simple shell script in `./vagrant`? Windows?

Reply:
This allows more flexibility in that the specific servers / profile can be specified programmatically.